data-collection/deploy/module-rds-usage.yaml

```diff
@@ -205,18 +205,18 @@ Resources:
                   rds_inventory.append(format_rds(rds))
               return rds_inventory
 
-          def store_data_to_s3(data, region, service, path, resource_type, resource_value, filename, accountID, payer_id):
+          def store_data_to_s3(data, region, service, path, resource_value, filename, accountID, payer_id):
               local_file = f"/tmp/{region}-{filename}"
               with open(local_file, 'w') as f:
                   json.dump(data, f, default=str)
                   f.write('\n')
               if os.path.getsize(local_file) == 0:
                   logger.info(f"No data in file for {path}")
                   return
-              key = f"{prefix}/{path}/payer_id={payer_id}/accountid={accountID}/region={region}/{resource_type}={resource_value}/year={year}/month={month}/{today}-{filename}"
```

Comment:
we are losing {resource_type}={resource_value}/ parition?

Reply:
The API returns it already so it is in the table. This resource id in the partition path is also not aligned to the other modules.